happiness_app/static/happiness_app/css/teampage_about.css

```diff
@@ -0,0 +1,35 @@
+/* css styles for team page and about section */
+
+
+.container {
+    
+}
```

Comment:
Suggestion (non-blocking): You can remove any empty style rules from CSS files. It won't break anything, but it is just good practice.